functions/arrow-functions.md

```diff
@@ -0,0 +1,239 @@
+| `}`                                                 | A closing curly brace to close the function body.                                                                                                                         |
+| `console.log(sayHello());`                          | We continue to invoke the `sayHello` function the same way we did other functions: add parentheses `()` next to the function name.                                        |
+
+### !callout-info
+
+## Arrow Functions Are Particular About Line Breaks
+
+Note that the arrow `=>` must be on the same line as the parameter list `()`. If they are on separate lines, it will generate a syntax error.
+
+### !end-callout
+
+## Syntax With One-Line Return Function Bodies
+
+Consider our `sayHello` arrow function:
+
+<!-- prettier-ignore-start -->
+```javascript
+const sayHello = () => {
+  return 'hey there.';
+}
+
+console.log(sayHello());
+```
+<!-- prettier-ignore-end -->
+
+This function is responsible for one thing: returning the string `'hey there.'`.
+
+When an arrow function is responsible for returning something, and has a function body that is one line long, we can use reduced arrow function syntax.
+
+<!-- prettier-ignore-start -->
+```js
+const sayHello = () => 'hey there.';
+```
+<!-- prettier-ignore-end -->
+
+| <div style="min-width:200px;"> Piece of Code </div> | Notes                                                                                                                                                                       |
+| --------------------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `const sayHello =`                                  | We declare the `const` variable `sayHello` and begin to assign it to an arrow function.                                                                                     |
+| `()`                                                | In one-line arrow functions, the parameter syntax stays the same.                                                                                                           |
+| `=>`                                                | In one-line arrow functions, we still need the arrow between the parameter list and the function body.                                                                      |
+| `'hey there.'`                                      | In one-line arrow functions, we can exclude the curly braces and the `return` keyword. However, this expression **_must_** be on the same line as the rest of the function. |
+| `;`                                                 | This statement now must conclude with a semicolon.                                                                                                                          |
```

Comment:
technically, the previous examples should also have used a semicolon, so this isn't really any different.
```suggestion
| `;`                                                 | This semicolon terminates the expression that assigns the arrow function to `sayHello`.                                                                                                                          |
```